core/include/ignition/plugin/config.hh

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef IGNITION_PLUGIN__CONFIG_HH_
+#define IGNITION_PLUGIN__CONFIG_HH_
+
+#include <gz/plugin/config.hh>
+
+/* Version number */
+// #define IGNITION_PLUGIN_MAJOR_VERSION GZ_PLUGIN_MAJOR_VERSION
+// #define IGNITION_PLUGIN_MINOR_VERSION GZ_PLUGIN_MINOR_VERSION
+// #define IGNITION_PLUGIN_PATCH_VERSION GZ_PLUGIN_PATCH_VERSION
+
+// #define IGNITION_PLUGIN_VERSION GZ_PLUGIN_VERSION
+// #define IGNITION_PLUGIN_VERSION_FULL GZ_PLUGIN_VERSION_FULL
+
+// #define IGNITION_PLUGIN_VERSION_HEADER GZ_PLUGIN_VERSION_HEADER
```

Comment:
We could remove these comments

Reply:
I'll probably leave them in for convenience if we ever end up tick-tocking the macros